### PR TITLE
helics up to 3.3.0

### PR DESCRIPTION
**Pull request by nikolayilyin**

### CODE REVIEW

File: src/main/shell/download_distributions.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+unpack_and_delete_archives() {
+  for f in *.zip; do unzip "$f"; done
+  for f in *.tar.gz; do tar -xvzf "$f"; done
+  # for f in *.gz; do gunzip "$f"; done
+  # for f in *tar; do tar -xvf "$f"; done
```
Comment (g-derzhavets) [inline src/main/shell/download_distributions.sh]: Why do we need this comments on line 6 and 7?

File: README.md [hunk 2]
```diff
@@ -9,17 +9,15 @@
 ```
 
 ### Update library
-1. Download distributions from release page, for example: https://github.com/GMLC-TDC/HELICS/releases/tag/v2.4.1. Not shared ones!
-2. Go to `java` folder and get copy new versions of `helics.jar` to the `libs` folder of the project
-3. Zip all libraries from `share` and `java` and put them to resources with related naming.  
+1. Download distributions from release page with a shell script - `src/main/shell/download_distributions.sh`
+2. Copy `helics.jar` to the `libs` folder of the project. The jar file might be copied from macOS, win32, win64 distributions (from java folder) or might be built with docker image together with libhelicsJava.so.
```
Comment (g-derzhavets) [inline README.md]:
```suggestion
2. Copy `helics.jar` to the `libs` folder of the project. The jar file might be copied from macOS, win32, win64 distributions (from java folder) or might be built with docker image together with `libhelicsJava.so`.
```

**Review by g-derzhavets — CHANGES_REQUESTED** [review]
LGTM, 2 minor comments/suggestions. Also, I tried to run the `HelicsExample` and it is kinda stuck with this log

```
/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/bin/java -javaagent:/Applications/IntelliJ IDEA CE.app/Contents/lib/idea_rt.jar=63567:/Applications/IntelliJ IDEA CE.app/Contents/bin -Dfile.encoding=UTF-8 -classpath /Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/charsets.jar:/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/ext/cldrdata.jar:/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/ext/dnsns.jar:/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/ext/jaccess.jar:/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/ext/legacy8ujsse.jar:/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/ext/localedata.jar:/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/ext/nashorn.jar:/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/ext/openjsse.jar:/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/ext/sunec.jar:/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/ext/sunjce_provider.jar:/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/ext/sunpkcs11.jar:/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/ext/zipfs.jar:/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/jce.jar:/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/jfr.jar:/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/jsse.jar:/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/management-agent.jar:/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/resources.jar:/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre/lib/rt.jar:/Users/gregory/projects/helics-wrapper2/build/classes/java/main:/Users/gregory/projects/helics-wrapper2/build/resources/main:/Users/gregory/projects/helics-wrapper2/libs/helics.jar com.github.beam.HelicsExample
Loaded /var/folders/qj/tgl6z0_91g54xx8v08ymh0v80000gp/T/helics/libhelicsJava.dylib
HELICS loaded. Version: 3.3.0 (2022-09-15)
Broker is connected
Registered publication.
To enter execution mode we need to have python scripts manually running (beam_pydss_broker.py and site_power_controller_federate.py).Entering execution mode ...

```

**Review by g-derzhavets — APPROVED** [review]

File: src/main/shell/download_distributions.sh (hunk 1 above)
Comment (nikolayilyin) [reply src/main/shell/download_distributions.sh]: Good catch!

**Pull request merged**